index.js

```diff
@@ -131,9 +179,19 @@ module.exports = options => {
     }
 
     const commandWithSemi = command + (command.endsWith(';') ? '' : ';')
+    const commandWithoutSemi = command.endsWith(';') ? command.substring(0, command.length - 1) : command
     return evalSol(commands.concat(commandWithSemi), options)
       .then(result => {
         commands.push(commandWithSemi)
+        if (Object.keys(specialCommands).indexOf(commandWithoutSemi) !== -1) {
+          let struct = {}
+          let results = result.toString().split(",")
+          let props = specialCommands[commandWithoutSemi]
+          for (let i = 0; i < props.length; i++) {
+            struct[props[i].prop] = results[i]
```

Comment:
Another way to do this would be with `reduce` instead of a `for` loop, but I don't have a strong preference here.